interpretation/ww_head.py

```diff
@@ -166,14 +167,31 @@ def process(message, isGameMaster = False):
         # Says the user must be in a cc if they're not.
         if is_command(message,['info']):
             guild = message.channel.guild
-            owner_id = channel_get(message.channel.id,'owner')
-            owner_object = guild.get_member(int(owner_id))
+            try:
+                owner_id = channel_get(message.channel.id,'owner')
+            except:
+                return[Mailbox().respond('Sorry, but it doesn\'t look like you\'re in a CC! If you are, please alert a Game Master as soon as possible.')]
```

Comment:
Why are you doing this? This function isn't supposed to raise an error. If it does, we should be aware if that, for the error may not occur in the field!